hack/build-local-images.py

```diff
@@ -166,17 +176,18 @@ def add_to_context(context_dir, source, destination, container_destination):
     sytem at the correct destination.
     """
     debug("Adding file:\n\tfrom {}\n\tto {}\n\tincluding in container at {}".format(
-    	source,
-    	join(context_dir, destination),
-		container_destination)
-   	)
+        source,
+        join(context_dir, destination),
+        container_destination)
+    )
     absolute_destination = abspath(join(context_dir, destination))
     if isdir(source):
         dir_util.copy_tree(source, absolute_destination)
     else:
         copy(source, absolute_destination)
     with open(join(context_dir, "Dockerfile"), "a") as dockerfile:
-        dockerfile.write("ADD {} {}\n".format(destination, container_destination))
+        dockerfile.write("ADD {} {}\n".format(destination,
```

Comment:
please no

Reply:
This is more pep8 compliance. There are multiple accepted styles, so are you more of a fan of the hanging indentation?

Reply:
lets just not make changes if we don't NEED to... you're not touching this code otherwise